DDF.xml

```diff
@@ -6114,7 +6114,21 @@ The Object also allows steering the device towards a particular PLMN, or, to a p
         <Owner>Vodafone</Owner>
         <Source>2</Source>
         <Ver>1.0</Ver>
-        <DDF>10481.xml</DDF>
+        <DDF>version_history/10481-1_0.xml</DDF>
+        <Vorto></Vorto>
+        <DDFLink>1</DDFLink>
+        <TS></TS>
+        <TSLink>0</TSLink>
+    </Item>
+    <Item>
+        <ObjectID>10481</ObjectID>
+        <URN>urn:oma:lwm2m:x:10481:2.0</URN>
+        <Name>TBD</Name>
+        <Description>Reservation 10481-v2.0</Description>
+        <Owner>TBD</Owner>
+        <Source>2</Source>
+        <Ver>2.0</Ver>
+        <DDF></DDF>
         <Vorto></Vorto>
         <DDFLink>1</DDFLink>
```

Comment:
@gocadimic In `prod,` if the object is the object file is not present, then the `DDFLink` must be set to empty. Otherwise, it will fail validation. In the placeholder branch, you can change that value to `1` to facilitate the validation as soon as the user uploads the object.

Reply:
@jpradocueva,
So, for now, should `DDFlink` be empty? Once the user uploads the object, the DDFlink needs to be `1`.